src/main/java/uk/gov/register/presentation/resource/ThrowableExceptionMapper.java

```diff
@@ -30,6 +31,12 @@ public Response toResponse(Throwable exception) {
             throw (ClientErrorException) exception;
         }
 
+        if (exception instanceof ParamException.PathParamException) {
```

Comment:
There is a functional test for this `EntryResourceFunctionalTest.return404ResponseWhenEntryNumberIsNotAnIntegerValue()`.